core/templates/pages/thanks-page/thanks-page.component.spec.ts

```diff
@@ -16,29 +16,28 @@
  * @fileoverview Unit tests for thanks page component.
  */
 
-// TODO(#7222): Remove the following block of unnnecessary imports once
-// App.ts is upgraded to Angular 8.
-import { UpgradedServices } from 'services/UpgradedServices';
-// ^^^ This block is to be removed.
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
 
-require('pages/thanks-page/thanks-page.component.ts');
+import { ThanksPageComponent } from './thanks-page.component';
+import { UrlInterpolationService } from
+  'domain/utilities/url-interpolation.service.ts';
 
 describe('Thanks page', function() {
-  var ctrl = null;
+  let component: ThanksPageComponent;
+  let fixture: ComponentFixture<ThanksPageComponent>;
 
-  beforeEach(angular.mock.module('oppia', function($provide) {
-    var ugs = new UpgradedServices();
-    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
-      $provide.value(key, value);
-    }
-  }));
+  beforeEach(() => {
```

Comment:
Use async, compile components is an async feature.
<img width="1440" alt="Screenshot 2020-07-14 at 10 54 32 AM" src="https://user-images.githubusercontent.com/23002114/87387463-6826fe00-c5c0-11ea-9918-76b8da7cfa7e.png">

Reply:
Done